packages/core/src/indexing/sqlite/sqlite-index-api.ts

```diff
@@ -26,22 +26,39 @@ export class SqliteIndexApi implements DatabaseIndexApi {
     this.insertionOrder = new InsertionOrder(dbConnection)
   }
 
+  public getActiveModelsToIndex(): Array<StreamID> {
+    // Helper function to get array of active models that are currently being indexed by node
+    // TODO: update to runtime check once adminAPI unlocks to load models on the fly
+    return this.modelsToIndex
+  }
+
+  private _isIndexStream(args: StreamID): Boolean {
+    // TODO: find more efficient way
```

Comment:
can use a Set instead of an array for `modelsToIndex`

Reply:
You could be surprised (I was), but Set is significantly slower than an array for <10000 entries. It is more idiomatic, but slow.

Reply:
yeah, that actually does make sense and is a fair point.  At my last job we used to have a type that implemented a `Set` API but used an array under-the-hood as the implementation, and we used that for small sets for exactly this reason.  I guess for now it's fine to just stick with Array.  But then let's remove that `TODO` comment too.